inductiva/admin/tasks.py

```diff
@@ -0,0 +1,43 @@
+"""Admin functions to manage tasks."""
```

Comment:
manage or query? This only provides the ability to check data and not add? Or in the future we will have that capability here?